examples/OECC/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenEdge Command Center #
+
```

Comment:
This should have some kind of summary or leading statement to provide context on what this is configuring. Even if something basic to start like "This will install the OECC in a Docker container" to a few additional lines that explains the purpose just so people understand what they're about to do via the instructions.

Reply:
Added introduction to README.md.